### ISSUE TITLE
this project is dead, who taking over - where is new project?

### ISSUE THREAD

**Issue by Jossnaz** [Haehnchen/idea-php-laravel-plugin#269]
this project is dead, who taking over - where is new project?

**Comment by mfn** [comment]
You might be interested in checking out https://plugins.jetbrains.com/search?products=phpstorm&search=laravel

**Comment by Jossnaz** [comment]
thanks... I only see that 50$ per year laravel plugin as alternative, you referring to that? did the owner turn it into a pay 2 play plugin?

**Comment by adelf** [comment]
It's $39 per year. No, it's not the author. I wrote a new Laravel plugin from scratch. There is a 30 day trial to try.

**Comment by nicolus** [comment]
It seems @Haehnchen is focusing exclusively on the Symfony plugin nowadays, which is a shame for us Laravel users :-/

I have switched to the paid laravel-idea plugin, but I still think it's important to have a free alternative. I was able to convince my hierarchy of the need to pay for PhpStorm without too much trouble, but explaining that we now need a plugin for that for the whole team will be a lot harder, so while I'm happy to pay for laravel-idea for my personal use, I'd prefer if my team was able to use a free and up to date plugin.

**Comment by onlime** [comment]
@adelf does your Laravel Idea plugin still require [barryvdh/laravel-ide-helper](https://github.com/barryvdh/laravel-ide-helper)? Does having enabled laravel-ide-helper give us any advantage / extra features or would it rather conflict with your plugin?
Does your plugin support Facades and Eloquent Query Scopes for code hinting out of the box?

Could you maybe add this to your FAQ, as I think a lot of idea-php-laravel-plugin users have laravel-ide-helper in place which is rather a workaround in my eyes. If your plugin does all this without any extra dev requirement, it's well worth it's price. But please make it more clear in your FAQ.

**Comment by adelf** [comment]
@onlime you’re right. My documentation is poor. Laravel Idea generates Facades, Macro and Eloquent helper code and provides “PhpStorm meta” file for all framework file, so I can say “laravel-ide-helper” package is not needed for 99% of projects. I’ll definitely update my docs. Thank you.

**Comment by jonnywilliamson** [comment]
@adelf Can I ask then another question leading on from that.

What if I have BOTH installed? Does laravel-ide-helper conflict or get in the way of your plugin?


Another "issue" i have is that I do this as a hobby, in my free time. I might code away on a project for 2 months then stop for number of weeks/months (i'm not a programmer in my job).

Can I only pay for the months that I need it? Ie, can I stop and start the monthly charge for each month that I might want to use it?

Thank you.

**Comment by adelf** [comment]
@jonnywilliamson yes. a monthly subscription is available. $3.9/month.

**Comment by Webinx** [comment]
It's definitively a dead project.
Hope someone will take over, Laravel deserves an open source plugin for PhpStorm.
